### ISSUE TITLE
Version 1.4.2 was built with UCS-2

### ISSUE THREAD

**Issue by NadavkOptimalQ** [confluentinc/confluent-kafka-python#882]
Description
===========
I Installed the package today and I got an error when I imported the consumer I got the error:
```Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/tmp/.pyenv/local/lib/python2.7/site-packages/confluent_kafka/__init__.py", line 19, in <module>
    from .deserializing_consumer import DeserializingConsumer
  File "/tmp/.pyenv/local/lib/python2.7/site-packages/confluent_kafka/deserializing_consumer.py", line 19, in <module>
    from confluent_kafka.cimpl import Consumer as _ConsumerImpl
ImportError: /tmp/.pyenv/local/lib/python2.7/site-packages/confluent_kafka/cimpl.so: undefined symbol: PyUnicodeUCS2_FromObject
```
I'm running on Ubuntu 18.04 with python 2.7.17. 
Tried this on python 3.6 and it didn't reproduce 

How to reproduce
================
`pip install confluent-kafka`
`python`
`from confluent_kafka import Consumer`

**Comment by AlexandreYang** [comment]
Same error here with Python 2.7: https://dev.azure.com/datadoghq/integrations-core/_build/results?buildId=16488&view=logs&j=ac360893-7fa6-510e-0d86-2e48566d2d4f&t=0c609eef-5895-5b89-9a57-a01f12a7bb14

```
E   CheckException: confluent_kafka was not imported correctly, make sure the library is installed and that you've set LD_LIBRARY_PATH correctly. Please refer to datadog documentation for more details. Error is /home/vsts/work/1/s/mapr/.tox/py27/lib/python2.7/site-packages/confluent_kafka/cimpl.so: undefined symbol: PyUnicodeUCS2_FromObject
```

**Comment by rnpridgeon** [comment]
Thank you both for reporting this. It would appear that test-manylinux.sh failed but ultimately it was ignored allowing the travis build to report green. Looking into fixing the packaging validation tests then I'll fix the build and push 1.4.3.

**Comment by rnpridgeon** [comment]
Notes for tracking:

Seems pip's new build in place functionality results in the reuse of build artifacts which were previously built (and linked to ucs2). 

https://github.com/pypa/pip/issues/8165
https://github.com/pypa/manylinux/pull/560

The problem isn't present in Python 3.6 because Python 3.3+ support ucs4 by default as documented in PEP 513. 

https://www.python.org/dev/peps/pep-0513/#ucs-2-vs-ucs-4-builds
